ch_backup/backup/deduplication.py

```diff
@@ -284,29 +284,30 @@ def collect_dedup_references_for_backup_deletion(
     """
     dedup_refences = collect_dedup_references_for_batch_backup_deletion(
         layout=layout,
-        retained_backups_with_light_meta=retained_backups_with_light_meta,
-        deleting_backups_with_light_meta=[deleting_backup_with_light_meta],
+        retained_backups=retained_backups_with_light_meta,
```

Comment:
Do we need method `collect_dedup_references_for_backup_deletion`?

Seems like it is only one method with `backups_with_light_meta=True`, except unittests - maybe we do not need this parameter at all?

Reply:
> Do we need method collect_dedup_references_for_backup_deletion?
Seem like not, it is unused method. Will remove it. Well, therefore we can remove `backups_with_light_meta`.

Reply:
Done